src/init.cpp

```diff
@@ -608,7 +608,7 @@ void SetupServerArgs(NodeContext& node)
     argsman.AddArg("-whitebind=<[permissions@]addr>", "Bind to given address and whitelist peers connecting to it. "
         "Use [host]:port notation for IPv6. Allowed permissions are bloomfilter (allow requesting BIP37 filtered blocks and transactions), "
         "noban (do not ban for misbehavior), "
-        "forcerelay (relay even non-standard transactions), "
+        "forcerelay (relay transactions that are already in the mempool; implies relay), "
```

Comment:
17985 what would it take to backport the test changes? https://github.com/bitcoin/bitcoin/pull/17985/files#diff-7d84d0ec5de995664cadef6f12ca42171b2fbe08c7539d5715b0066b9ca7ea10

Reply:
So there was a funny little commit known as 17984 that i needed to backport, sbf now